### ISSUE TITLE
training loss/testing top-1

### ISSUE THREAD

**Issue by HaichenHe** [HJYao00/Side4Video#7]
Hi! Thank you for your excellent work. When I try to reproduce your code on k400(ViT-B/16-8f), after 3 epochs there is a situation where the training set loss is decreasing, but the test set accuracy is also decreasing dramatically(61.3->67.2->67.9->66.9->61.9). Has this situation happened to you? Can you please provide the training log files?

**Comment by HJYao00** [comment]
I'm sorry, I haven't encountered this situation before. Please make sure that the batch size and other hyperparameters are consistent. Could you send me an email? I'll share the log files with you. This is my email: yaohj22@mails.tsinghua.edu.cn

**Comment by HaichenHe** [comment]
Thankyou!
